docs/heap/Kth-largest-using-min_heap.md

```diff
@@ -0,0 +1,147 @@
+---
+id: Kth-Largest-Element-MinHeap
+title: Kth Largest Element using Min Heap
+sidebar_label: Kth Largest Element
+sidebar_position: 3
+description: Finding the Kth largest element in an array using a min heap data structure, commonly used in streaming and top-K problems.
+tags: [Heap, Priority Queue, Top-K, Streaming]
+---
+
+# Kth Largest Element Using Min Heap
+
+A **Min Heap** based approach to find the Kth largest element in an unsorted array. This algorithm maintains a min heap of size K to efficiently track the K largest elements, where the root of the heap represents the Kth largest element.
+
+## Introduction
+
+The K-th largest element algorithm using a min heap is particularly useful when dealing with:
+- Large datasets where we need to find top K elements
+- Streaming data where elements arrive continuously
+- Memory-constrained environments where we can't sort the entire array
+
+The key idea is to maintain a min heap of size K, where the smallest element (root) represents the Kth largest element seen so far.
+
+## Algorithm Overview
+
+### Basic Concept
+
+1. Create a min heap to store the K largest elements
+2. Process elements one by one:
+   - If heap size < K: Insert the element
```

Comment:
replace `<` with ``` `<` ``` or `&lt;` because our site is based on markdown (`.md` / `.mdx`) + React (`.js` / `.jsx`) so, you can't use HTML tags and jsx syntax in .md file

Reply:
done